### PR TITLE
[DEV-7101] Using new Preview Images 

### PR DESCRIPTION
**Pull request by MaxwellKendall**
**High level description:**

Using new preview images.

**Technical details:**

We were actually relying on the image being defined in index.ejs to some degree, so I ended up deleting some of that code in the index.ejs file and adding some new stuff to the MetaTags component where we dynamically define all this. 

Ideally, we should've just had to update the data mapping / helper file.

**JIRA Ticket:**
[DEV-7101](https://federal-spending-transparency.atlassian.net/browse/DEV-7101)

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
`N/A` Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
`N/A` Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
`N/A` Verified mobile/tablet/desktop/monitor responsiveness
`N/A` Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
`N/A` Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
`N/A` [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
`N/A` [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
`N/A` Design review complete `if applicable`
`N/A` [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [x] Code review complete

### CODE REVIEW

**Comment by MaxwellKendall** [comment]
image

**Comment by MaxwellKendall** [comment]
image

File: src/js/helpers/metaTagHelper.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 // Disabling max-len property for readability / editability
 /* eslint-disable max-len */
 
-const productionURL = 'https://usaspending.gov/';
+const productionURL = 'https://sandbox.usaspending.gov/';
```
Comment (brett-varney) [inline src/js/helpers/metaTagHelper.js:8]: Should this still be "sandbox"?

**Review by brett-varney — CHANGES_REQUESTED** [review]
One code question, and are there updates for state and recipient pages?

**Review by brett-varney — APPROVED** [review]
I guess it works... I can't test it myself. No new errors

**Pull request merged**